tests/unit_tests/commandline/test_geoips_validate.py

```diff
@@ -0,0 +1,92 @@
+"""Unit test for GeoIPS CLI `validate` command.
+
+See geoips/commandline/ancillary_info/cmd_instructions.yaml for more information.
+"""
+
+from glob import glob
+from importlib import resources
+import pytest
+
+from tests.unit_tests.commandline.cli_top_level_tester import BaseCliTest
+
+
+class TestGeoipsValidate(BaseCliTest):
+    """Unit Testing Class for GeoipsValidate Command."""
+
+    @property
+    def all_possible_subcommand_combinations(self):
+        """A list of every possible call signature for the GeoipsValidate command.
+
+        This includes failing cases as well.
+        """
+        if not hasattr(self, "_cmd_list"):
+            self._cmd_list = []
+            base_args = self._validate_args
+            pkg_path = str(resources.files("geoips_clavrx") / "plugins")
```

Comment:
inconsistent naming convention, elsewhere it's `_package`

Reply:
still unfixed in revisions

Reply:
This was updated to `pkg_path` in other locations rather than `_package_path`. Naming is consistent now.